src/node_blob.cc

```diff
@@ -47,7 +47,7 @@ void Blob::Initialize(
   SetMethod(context, target, "storeDataObject", StoreDataObject);
   SetMethod(context, target, "getDataObject", GetDataObject);
   SetMethod(context, target, "revokeDataObject", RevokeDataObject);
-  FixedSizeBlobCopyJob::Initialize(env, target);
+  //FixedSizeBlobCopyJob::Initialize(env, target);
```

Comment:
Nit: leave a TODO notice in these places where you comment out lines that we'll need to go back and cleanup